ixx/src/action/index.rs

```diff
@@ -67,22 +98,42 @@ fn into_option(name: &str, option: option::Option) -> libixx::Option {
   }
 }
 
-static THEME_SET: LazyLock<syntect::highlighting::ThemeSet> =
-  LazyLock::new(ThemeSet::load_defaults);
-static SYNTAX_SET: LazyLock<SyntaxSet> = LazyLock::new(SyntaxSet::load_defaults_newlines);
-
-fn code_highlighter(code: &str, lang: &str) -> String {
-  let syntax = if let Some(syntax) = SYNTAX_SET.find_syntax_by_name(lang) {
-    syntax
-  } else {
-    SYNTAX_SET.find_syntax_by_extension("html").unwrap()
-  };
-
-  highlighted_html_for_string(
-    code,
-    &SYNTAX_SET,
-    syntax,
-    &THEME_SET.themes["InspiredGitHub"],
+fn update_declaration(declaration: Declaration) -> String {
+  // NOTE: Is the url actually optional? If its true, this can be ignored.
+  //       Otherwise the fallback with building the url outself is required.
+  //
+  // if "url" in declaration:
+  //   return declaration["url"]
```

Comment:
I am not sure if that is specific to eg nixvim or not anymore

Reply:
I would like to test it, let's merge it and check.